lib/semantics/check-omp-structure.h

```diff
@@ -21,7 +21,9 @@
 #define FORTRAN_SEMANTICS_CHECK_OMP_STRUCTURE_H_
 
 #include "semantics.h"
+#include "tools.h"
```

Comment:
This include isn't needed here, only in the .cc file.

Reply:
Yes, I forgot to change that after moving around those functions. Thanks.

Reply:
Addressed.